packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -161,6 +192,24 @@ export class FileNavigatorContribution extends AbstractViewContribution<FileNavi
         };
         updateFocusContextKeys();
         this.shell.activeChanged.connect(updateFocusContextKeys);
+        this.workspaceCommandContribution.onDidCreateNewFile(async uri => this.onDidCreateNewResource(uri));
+        this.workspaceCommandContribution.onDidCreateNewFolder(async uri => this.onDidCreateNewResource(uri));
+    }
+
+    private async onDidCreateNewResource(uri: URI): Promise<void> {
+        const navigator = await this.tryGetWidget();
```

Comment:
`await` is not necessary

btw Theia should alert about it

Reply:
fixed